docs/handbook/text-anchors.rst

```diff
@@ -0,0 +1,140 @@
+
+.. _text-anchors:
+
+Text anchors
+============
+
+The ``anchor`` parameter determines alignment of drawn text relative to the ``xy`` parameter.
+The default alignment is top left, specifically ``la`` (left--ascender) for horizontal text
+and ``lt`` (left--top) for vertical text.
```

Comment:
Nit: is there a reason for the double dash?

```suggestion
and ``lt`` (left--top) for vertical text.
```

Reply:
I wanted to somehow make it clear that e.g. "left-ascender" means left horizontally and ascender vertically, and an en-dash is sometimes used in complex situations in place of a hyphen, but I think this is more confusing then helpful. I'll change these to hyphens, i.e. `-`.